src/Action/Server.hs

```diff
@@ -79,6 +83,7 @@ actionReplay Replay{..} = withBuffering stdout NoBuffering $ do
             evaluate $ rnf res
             putChar '.'
     putStrLn $ "\nTook " ++ showDuration t ++ " (" ++ showDuration (t / intToDouble (repeat_ * length qs)) ++ ")"
+actionReplay _ = error "should not happen"
```

Comment:
Before, if we got here it would print an error message including the line number and everything. Now, we're relying on stack traces that are much more fickle.